include/RAJA/policy/loop/scan.hpp

```diff
@@ -51,10 +51,11 @@ inclusive_inplace(
     Iter end,
     BinFn f)
 {
-  auto agg = *begin;
+  using ValueT = typename std::remove_reference<decltype(*begin)>::type;
+  ValueT agg = *begin;
 
   for (Iter i = ++begin; i != end; ++i) {
-    agg = f(*i, agg);
+    agg = f(agg, *i);
```

Comment:
what's going on with the transposed function args here and below?

Reply:
I don't know why this discrepancy existed, but the new order seems more right to me, though though it shouldn't matter as our operator should be commutative.

Reply:
At any rate, good catch! 👍

Reply:
I looked at the blame history and it was my fault :\ dunno what I was doing there.